### PR TITLE
gcp-lts-bom: clearing comments on BOMs

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

File: boms/cloud-lts-bom/pom.xml [hunk 1]
```diff
@@ -288,7 +288,6 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-bigquerystorage-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:291]: @stephaniewang526 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 2]
```diff
@@ -299,13 +298,11 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-trace-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:301]: @Neenu1995 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 3]
```diff
@@ -299,13 +298,11 @@
...
         <version>${google-cloud-trace.version}</version>
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-monitoring-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:306]: @Neenu1995 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 4]
```diff
@@ -326,7 +323,6 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-tasks-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:326]: @Neenu1995 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 5]
```diff
@@ -337,6 +333,7 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
+        <!-- Not using a BOM because it has google-cloud-bigtable-emulator -->
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:336]: @kolea2 FYI, I added comment about not using google-cloud-bigtable-bom.

File: boms/cloud-lts-bom/pom.xml [hunk 6]
```diff
@@ -367,31 +364,26 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-service-usage-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:367]: @Neenu1995 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 7]
```diff
@@ -367,31 +364,26 @@
...
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-container-bom</artifactId>
         <version>${google-cloud-container.version}</version>
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-orchestration-airflow-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:377]: @eaball35 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 8]
```diff
@@ -367,31 +364,26 @@
...
         <version>${google-cloud-orchestration-airflow.version}</version>
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-resourcemanager-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:382]: @eaball35 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 9]
```diff
@@ -367,31 +364,26 @@
...
         <version>${google-cloud-resourcemanager.version}</version>
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-redis-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:387]: @eaball35 Can I use this BOM to cover API surface?

File: boms/cloud-lts-bom/pom.xml [hunk 10]
```diff
@@ -367,31 +364,26 @@
...
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-service-usage-bom</artifactId>
         <version>${google-cloud-service-usage.version}</version>
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-container-bom</artifactId>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:372]: @shabirmean Can I use this BOM to cover API surface?

**Review by Shabirmean — APPROVED** [review]
[Version is `2.2.1`](https://github.com/googleapis/java-container/blob/v2.2.1/google-cloud-container-bom/pom.xml)  (`<google-cloud-container.version>2.2.1</google-cloud-container.version>`) for `java-container` lib
LGTM!

File: boms/cloud-lts-bom/pom.xml (hunk 1 above)
Comment (stephaniewang526) [reply boms/cloud-lts-bom/pom.xml:291]: Yes.

File: boms/cloud-lts-bom/pom.xml (hunk 5 above)
Comment (kolea2) [reply boms/cloud-lts-bom/pom.xml:336]: Thanks!

**Review by chanseokoh — APPROVED** [review]

File: boms/cloud-lts-bom/pom.xml (hunk 2 above)
Comment (Neenu1995) [reply boms/cloud-lts-bom/pom.xml:302]: yes

File: boms/cloud-lts-bom/pom.xml (hunk 3 above)
Comment (Neenu1995) [reply boms/cloud-lts-bom/pom.xml:309]: yes

File: boms/cloud-lts-bom/pom.xml (hunk 4 above)
Comment (Neenu1995) [reply boms/cloud-lts-bom/pom.xml:331]: yes

File: boms/cloud-lts-bom/pom.xml (hunk 6 above)
Comment (Neenu1995) [reply boms/cloud-lts-bom/pom.xml:376]: yes

**Comment by suztomo** [comment]
Got OK from Emily.

**Pull request merged**